docs/demos/index.txt

```diff
@@ -260,6 +260,21 @@ Find a demo for an algorithm
 
 See :doc:`the root README <../README>` or each algorithm's documentation for the relevant citation(s).
 
+
+Download the demos
+------------------
+
+You can run download a local copy of the demos using the :code:`curl` command below:
```

Comment:
No change required, but for future reference, I think code/literal formatting can be more conveniently expressed with just double backticks?

```suggestion
You can run download a local copy of the demos using the ``curl`` command below:
```

(I guess it's slightly different, in that the `:code:` role ones end up with the `code` CSS class in addition to `docutils literal notranslate`. 🤷‍♂️ )

Reply:
Ah thanks, yeah I'm still having to google every little thing I wanna do in rst